### DISCUSSION TITLE
Error with a script

### DISCUSSION THREAD

**Discussion by DuttaAbhi02** [satijalab/seurat#8157] [General]
Can someone please help to overcome the following error!!!

nonCM_MI_D7_1 <- FindNeighbors(nonCM_MI_D7_1, dims = 1:10)
Computing nearest neighbor graph
Computing SNN
Error in validObject(.Object) : 
  invalid class “Graph” object: superclass "mMatrix" not defined in the environment of the object's class

**Comment by MicChen98** [comment]
I meet the same error! Have you overcame the issue?
